yorkie/src/main/kotlin/dev/yorkie/document/crdt/Rht.kt

```diff
@@ -0,0 +1,50 @@
+package dev.yorkie.document.crdt
+
+import dev.yorkie.document.time.TimeTicket
+import dev.yorkie.document.time.TimeTicket.Companion.compareTo
+
+/**
+ * [Rht] is a replicated hash table by creation time.
+ * For more details about RHT:
+ * @link http://csl.skku.edu/papers/jpdc11.pdf
+ */
+internal class Rht : Iterable<Rht.Node> {
+    private val nodeMapByKey = mutableMapOf<String, Node>()
+
+    val entrySet: List<Entry>
```

Comment:
I converted the below original function `toObject()` from JS-SDK to `entrySet` in Android-SDK, but I'm not sure if this is the best way to handle it.

```js
  /**
   * `toObject` returns the object of this hashtable.
   */
  public toObject(): Record<string, string> {
    const obj: Record<string, string> = {};
    for (const [key, node] of this.nodeMapByKey) {
      obj[key] = node.getValue();
    }

    return obj;
  }
```

Reply:
I think property name should be `entries` or return `Set`.